packages/malloy/src/lang/ast/types/malloy-element.ts

```diff
@@ -333,6 +334,10 @@ export class RunList extends ListOf<DocStatement> {
   executeList(doc: Document): ModelDataRequest {
     while (this.execCursor < this.elements.length) {
       if (doc.errorsExist()) {
+        // TODO make a better way to stop cascading errors -- this way means that if there are
+        // actual different errors in multiple places, we only see the first one. A better way
+        // might be to make things that have error pass all kinds of typechecks, which prevents
+        // the real cascade.
```

Comment:
I think this is a separate project — If instead we just have the `unknown` type (maybe rename to `error` type) be accepted everywhere instead of rejected everywhere, I think we remove a ton of cascading for cases that are real cascading issues without removing totally unrelated error messages.